packages/sdk/src/actions/listToken.ts

```diff
@@ -12,7 +12,7 @@ type ListTokenBody = NonNullable<
 
 type Data = {
   listings: Required<ListTokenBody>['params']
-  signer: Signer
+  signer: WalletClient
```

Comment:
should we rename the key here to be wallet or something similar?

Reply:
Yeah that's a good thought. Pedro and I talked about renaming it briefly. What do you think @pedromcunha, should we update it?

Reply:
Idk, feels like it's clearly established what a signer is and to change it to wallet may lead to some confusion.